src/edu/washington/escience/myria/operator/Difference.java

```diff
@@ -163,7 +161,7 @@ public void init(final ImmutableMap<String, Object> execEnvVars) throws DbExcept
       throw new DbException("Incompatible input schemas");
     }
 
-    tupleIndices = new TIntObjectHashMap<TIntList>();
+    tupleIndices = new IntObjectOpenHashMap<IntArrayList>();
```

Comment:
Would prefer `new IntObjectOpenHashMap<>()` here. `javac` will infer the correct generic type, and then there's less work to refactor in the future.

(This trick was discovered fairly late in Myria, so hasn't made it through much of the code.)